connectors/sources/tests/test_gcs.py

```diff
@@ -72,7 +72,7 @@ async def test_empty_configuration():
 
 
 @pytest.mark.asyncio
-async def test_ping_for_successful_connection():
+async def test_ping_for_successful_connection(catch_stdout, patch_logger):
```

Comment:
So I'm sure this is fine, but I'm curious; where do these arguments come from? Is this just part of the testing lib or something?

Reply:
yes, this is part of `pytest` fixtures.

every function decorated with `@pytest.fixture` in https://github.com/elastic/connectors-python/blob/main/connectors/conftest.py can be used in tests as arguments like this.

when they are added, the fixture is used as a wrapper :

```
def fixture():
    # do something before the test
    yield  # this calls the tests
    # do something after the test
```

Reply:
more on fixtures https://docs.pytest.org/en/6.2.x/fixture.html